tests/safety/common.py

```diff
@@ -288,6 +283,18 @@ def test_non_realtime_limit_up(self):
     self._set_prev_torque(0)
     self.assertFalse(self._tx(self._torque_cmd_msg(-self.MAX_RATE_UP - 1)))
 
+class SteerRequestCutTestBase(unittest.TestCase, abc.ABC):
+
+  # Safety around steering req bit
+  MIN_VALID_STEERING_FRAMES: Optional[int] = None
+  MAX_INVALID_STEERING_FRAMES: int = 1
+  MIN_VALID_STEERING_RT_INTERVAL: Optional[int] = None
+
+  def test_steer_req_parameters(self):
+    # assert the parameters are properly set
+    self.assertIsNotNone(self.MIN_VALID_STEERING_FRAMES)
+    self.assertIsNotNone(self.MIN_VALID_STEERING_RT_INTERVAL)
```

Comment:
I'd remove the checks below so things will fail if they are 0. Can you also make this a new PR?

Reply:
made a separate PR